composer.json

```diff
@@ -30,7 +30,7 @@
         "composer/installers": "^1.9",
         "cweagans/composer-patches": "^1.6.5",
         "dof-dss/nicsdru_nidirect_theme": "^0.12.1",
-        "dof-dss/nicsdru_origins_modules": "^0.9.13",
+        "dof-dss/nicsdru_origins_modules": "dev-D8NID-1105",
```

Comment:
I assume that this is a mistake and that you meant to change it back ?

Reply:
Yep, exactly right. I need this in place right now otherwise the project builds using the tag which means I can't combine it with the corresponding branch on the other repo.

Last time I forgot to update before merging so making an extra note to do it on this PR to save a hasty commit to follow it :)